### DISCUSSION TITLE
Week 6 - Use Git Effectively

### DISCUSSION THREAD

**Discussion by MichaelCade** [MichaelCade/90DaysOfDevOps#32] [General]
https://github.com/MichaelCade/90DaysOfDevOps#use-git-effectively

**Comment by Med-H** [comment]
Maybe add git rebasing ?

**Reply by MichaelCade** [reply]
Thanks, I actually add something on `git rebase` and `git merge` on Day 39 of the project. Let me know if there is something missing here, I know we can get into interactive rebasing which adds more complexity to the command set but I think for an overview I am hopeful I covered enough. 

Thanks for your comment though, really useful and a good reminder to include something.

**Reply by Med-H** [reply]
my bad :). I was thinking about interactive rebasing :)

**Reply by MichaelCade** [reply]
I will see if I can find some time tomorrow to write something up quickly.
